### PR TITLE
Add the nvidia xserver driver and config

### PR DESCRIPTION
**Pull request by jocado**
This can still be required for some use cases, and is something that nvidia container toolkit looks for to inject via CDI config.

### CODE REVIEW

**Review by Saviq — APPROVED** [review]
This looks fine to me, we'll follow up with the likely addition of `xerver-xorg-video-all`.

**Comment by jocado** [comment]
Thank you @Saviq :+1: 

I don't have permission to merge. Are you able to do that, or do I need a second review ?

**Comment by Saviq** [comment]
@jocado we have our kernel team looking at this, they'll get this in ASAP.

**Pull request merged**
